finder-app/finder-test.sh

```diff
@@ -54,7 +54,7 @@ fi
 
 for i in $( seq 1 $NUMFILES)
 do
-	./writer.sh "$WRITEDIR/${username}$i.txt" "$WRITESTR"
+	#./writer.sh "$WRITEDIR/${username}$i.txt" "$WRITESTR"
```

Comment:
@shpr8875 this doesn't look right, not sure you intended to make this change?  You should be invoking your cross compiled `./writer` application here.